Common/Core/TrackSelectionDefaults.cxx

```diff
@@ -107,4 +107,17 @@ TrackSelection getGlobalTrackSelectionRun3HF()
   return selectedTracks;
 }
 
+// Reduced default track selection for jet validation based on hybrid cuts for converted (based on ESD's from run 2) A02D's
+TrackSelection getJEGlobalTrackSelectionRun2()
+{
+  TrackSelection selectedTracks = getGlobalTrackSelection();
+  selectedTracks.SetTrackType(o2::aod::track::Run2Track); // Run 2 track asked by default
```

Comment:
This should not be needed, see e.g. https://github.com/AliceO2Group/O2Physics/pull/2508/files#diff-c06f7759449f2d85176eb11d4c829f7b3120cf29eec710f0060bd31a46a15c63R29